esphome/components/esp32_improv/esp32_improv_component.cpp

```diff
@@ -112,7 +112,7 @@ void ESP32ImprovComponent::loop() {
         this->set_state_(improv::STATE_AUTHORIZED);
       } else
 #else
-      this->set_state_(improv::STATE_AUTHORIZED);
+      { this->set_state_(improv::STATE_AUTHORIZED); }
```

Comment:
Maybe a little cleaner split lines?
```suggestion
      {
        this->set_state_(improv::STATE_AUTHORIZED);
      }
```

Reply:
That's what I had and the commit hooks changed it to this 🫤

Reply:
strange that it seemingly didn't for any of the other very similar cases 🤔

Reply:
Depends on exact line length I think